src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -205,6 +214,32 @@ def __init__(
                 + str(self._get_url())
             )
 
+    def _resolve_localhost(self):
+        global SERVER_IP
+
+        ipv6_localhost = "http://[::1]"
+        ipv4_localhost = "http://127.0.0.1"
+
+        if self._check_address_for_response(ipv6_localhost):
+            SERVER_IP = ipv6_localhost
```

Comment:
I can only manage to test ipv6 on my machine/in our unit tests but I have verified that ipv4 is correct in the past - I used this before I was using the new Ansys VPN. 

Also I think if both of these attempts are unsuccessful then it will revert to using "localhost". My main concern is that this will add a delay to the initial connection

Reply:
I think better to add later if needed.